### ISSUE TITLE
Game window after changed to fullscreen and windowed again

### ISSUE THREAD

**Issue by Redysz** [lordmauve/pgzero#233]
If I make game size fullscreen and then I again change to windowed, I won't have opportunity to move the window of a game.

A piece of code:
```
WIDTH = 800
HEIGHT = 600

FULLSCREEN = False
def on_key_down(key):
    global FULLSCREEN
    if key == keys.F:
        if FULLSCREEN:
            screen.surface = pygame.display.set_mode((WIDTH, HEIGHT), pygame.RESIZABLE)
        else:
            screen.surface = pygame.display.set_mode((1920, 1080), pygame.FULLSCREEN)
        FULLSCREEN = not FULLSCREEN
```

Then my game window is on top and left and I can't change it
![image](https://user-images.githubusercontent.com/25427869/99885942-02671480-2c39-11eb-97ab-ef0eac0590d1.png)

**Comment by patentfm** [comment]
I have the same issue +1

**Comment by fillwithjoy1** [comment]
@lordmauve 
Yep I've got this issue too

**Comment by lordmauve** [comment]
Does this still happen under PyGame 2.1.2?

**Comment by lordmauve** [comment]
This was fixed in Pygame 2.0.2: https://github.com/pygame/pygame/pull/2460

**Comment by fillwithjoy1** [comment]
> Does this still happen under PyGame 2.1.2

I haven’t used pgzero ever since my school assessment
